docs/monoDrive_home/Lidar.md

```diff
@@ -46,7 +96,8 @@ There are 12 data blocks, each block is **100 bytes** and there are **4 bytes**
 for the timestamp and **2 bytes** called factory bytes. The number of packets 
 for a revolution depends on the **horizontal revolution**.  
 
-### Visualizing LiDAR Output
+
+## Visualizing LiDAR Output
```

Comment:
We should add an image of the semantic lidar here.

![image](https://user-images.githubusercontent.com/6026642/86201592-97039580-bb25-11ea-95ae-fb145319d725.png)

and here's a matching picture of the regular lidar:

![image](https://user-images.githubusercontent.com/6026642/86201825-27da7100-bb26-11ea-9496-9465228c4828.png)

Reply:
@bradleynull these pictures look great on the documentation, thank you.